packages/vm/lib/evm/opcodes/functions.ts

```diff
@@ -940,6 +940,8 @@ export const handlers: Map<number, OpHandler> = new Map([
 
       const [value, offset, length] = runState.stack.popN(3)
 
+      accessAddressEIP2929(runState, runState.eei.getAddress(), false)
+
```

Comment:
Just as some post-review question: is the order with `subMemUsage()` relevant here? Since in all the other cases this is the other way around?

Reply:
Good question, and also a bit sloppy of me. But I just explicitly checked; the memory gas cost is deterministic and does not depend on the current gas available, so the order does not matter. I don't really want to trigger a new CI run, so I will merge here.